CONTRIBUTING.md

```diff
@@ -257,37 +235,20 @@ with a [GPG key](http://www.cryptol.net/files/Galois.asc).
 
 ## Cutting releases
```

Comment:
Also note that we now have https://gitlab-ext.galois.com/program-analysis/guidance/-/blob/master/HackageRelease.org and while this isn't a Hackage release, we might want to extend that documentation.  It's also not visible from outside the Galois network.

Reply:
Yeah, that's a good idea. My focus for these changes is on removing the `make`-based build system, rather than fully fleshing out the contribution documentation, but I think we should do a more thorough job of that soon, too.